src/PileupTrack.js

```diff
@@ -62,6 +61,65 @@ function makePath(scale, visualRead: VisualAlignment) {
   return d3.svg.line()(path);
 }
 
+// Create the SVG element for a single Cigar op in an alignment.
+function enterSegment(parentNode, op, scale) {
+  if (op.op == 'M') {
```

Comment:
Could be good to comment what 'M', 'D', etc are. I'm guessing match, deletion, insertion? Could also use const variables for them/remove magic strings.